sdk/reservations/Azure.ResourceManager.Reservations/src/autorest.md

```diff
@@ -57,8 +57,8 @@ directive:
       $.ReservationOrderResponse.properties.etag['x-ms-format'] = 'etag';
       $.PurchaseRequest['x-ms-client-name'] = 'PurchaseRequestContent';
       $.Price['x-ms-client-name'] = 'PurchasePrice';
-      $.Catalog.properties.resourceType['x-ms-format'] = 'resource-type';
+       $.Catalog.properties.resourceType['x-ms-client-name'] = 'reservedResourceType';
```

Comment:
remove the extra leading space

Reply:
also, any reason we change the type back to `string`?

Reply:
The reason that we revert this change is because the `resourceType` here in not a truly `ResourceType` that we defined in Azure.Core. It ‘s more like a simplified resource type. For instance, it will be `virtualMachines` instead of `Microsoft.Compute/virtualMachines`